birdbox/microsite/blocks.py

```diff
@@ -553,7 +553,7 @@ class Meta:
     def frontend_media(self):
         "Custom property that lets us selectively include CSS"
         return forms.Media(
-            css={"all": [static("css/birdbox-article.css")]},
+            css={"all": [static("css/protocol-article.css")]},
```

Comment:
Doesn't this mean we lose the bb-specific styling from `src/css/article.scss`? I think we'll be using that _somewhere_ in production, I imagine

Reply:
We can include both CSS files, if that's not gonna leave us with clashing classes. Indeed, I think it won't clash because the current stuff in `birdbox-article.css` is prefixed with `bb-`

Reply:
I think we can and should keep 'em both - protocol core stuff + bb-only customisation.

```suggestion
            css={"all": [static("css/protocol-article.css"), static("css/birdbox-article.css")]},
```

Reply:
The birdbox-article is being used for blog posts, and the regular Protocol article is being used for regular article blocks. I regret the confusing naming, and we could rename the blog one to something more descriptive like "blogpost" but that would require a migration.

Blog pages were new and non-Protocol so ended up getting their own style sheet, I just foolishly named it the same thing as the Protocol component, though with a different namespaced class.